cpp/test/CMakeLists.txt

```diff
@@ -219,38 +219,7 @@ if(BUILD_TESTS)
     NAME
     LINALG_TEST
     PATH
-    linalg/add.cu
```

Comment:
I think this file was accidentally checked in - we should probably keep the linalg/sparse tests here

Reply:
OMG, I forgot to recover the test cases a second time...T_T